tests/test_check.py

```diff
@@ -13,7 +13,7 @@
 def test_check_on_dir():
     """The function, check_all() should handle a top-level dir as input."""
     reporter = python_ta.check_all(
-        "examples",
+        "examples/sample_dir",
```

Comment:
Good! In addition to my comment about the `sample_dir` above, now that you've narrowed down the scope of the test, it would be good to `assert` that the filenames in the output match exactly what's expected (i.e., all files under `sample_dir`).